config/development.json

```diff
@@ -31,6 +31,8 @@
 		"S3_ACCESS_KEY": "S3RVER",
 		"S3_SECRET_KEY": "S3RVER"
 	},
+	"H5P_EDITOR": {
```

Comment:
It should not add for now. Only if we have really a need.

Reply:
Removed in favor of defaults